tests/amazon/test_credentials.py

```diff
@@ -33,8 +33,7 @@
     @mock_sts
     it "verify_creds ensures account_id matches aws":
         credentials = Credentials('us-west-1', 987654321, NotSpecified)
-        with self.assertRaises(BespinError):
-            credentials.verify_creds() # 987654321 != 123456789012
+        self.assertRaises(BespinError, credentials.verify_creds) # 987654321 != 123456789012
```

Comment:
I suggest 

   with self.fuzzAssertRaisesError(BespinError, "message you expect", kwarg1=1, kwarg2=2):
        credentials.verify_creds()

It's defined by delfick_error (https://github.com/delfick/delfick_error/blob/master/delfick_error.py#L143)